ddcz/html.py

```diff
@@ -250,3 +251,35 @@ def encode_valid_html(entity_string):
                 continue
 
     return encoded_safe_string
+
+
+class HtmlChecker(HTMLParser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.opened_tags = []
+
+    def handle_starttag(self, tag, attrs):
+        if tag not in ['br']:
```

Comment:
Why is only br in here? Don't we need also `hr` in here or `input`s?

Reply:
`hr` is a good possibility, but should be checked. Inputs are definitely not allowed within article text.